CHANGELOG.md

```diff
@@ -20,6 +18,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - feat: use updated ecs cluster source for ecs cluster
 - feat: consolidate ecs cluster module and ecs service module into one
 
+## [1.1.4] - 2022-10-10
+### Changes
+- fix: CKV_AWS_103: "Ensure that load balancer is using TLS 1.2"
+- feat: update files
+- feat: update github workflows
+- feat: Add supporting resources (vpc and ecs cluster)
+- feat: Make creation of KMS Key optional (i.e external KMS key can be used)
```

Comment:
hey @patrickmukumbu should we rename the branch name to have a feature prefix?

Reply:
Hey @ndegwajohn  since the scope was first to finish the fix I think it would just be great for it to remain that way